### PR TITLE
Feature/new react polymorph bubble component

### PR DESCRIPTION
**Pull request by bendyorke**
Starts work on #26.  It just creates the Bubble component, simple skin, theme, and stories.  @DominikGuzei Let me know if this is along the lines of what you had in mind, and any feedback you have.  If it looks good, I can use it in the Options component, and create the Tooltip component.

### CODE REVIEW

**Comment by bendyorke** [comment]
Ah, sorry, there was an issue with my monitor.  I was confused why I had to halve the border radius 😂

**Comment by bendyorke** [comment]
Latest version is looking correct for me on my laptop's display.  I've also included the work to use the `Bubble` component in the `Options` component - I've tested it in all of the stories for `Autocomplete` and `Select` and it works for me.  I'll work on the `Tooltip` component later tonight, let me know if you have any other items to note!

File: source/components/Autocomplete.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ export default class Autocomplete extends FormField {
     const { selectedOptions, filteredOptions, isOpen } = this.state;
     return Object.assign({}, super.prepareSkinProps(props), {
       selectedOptions,
-      filteredOptions,
+      filteredOptions: filteredOptions || [],
```
Comment (bendyorke) [inline source/components/Autocomplete.js]: @DominikGuzei It's worth pointing out that this is fixing a fatal bug, where pressing the backspace key while in an empty `Autocomplete` component would cause a crash.  I'm not sure if you'd rather me break this out to it's own branch or if this is fine.  It is mentioned in the git commit

Comment (DominikGuzei) [reply source/components/Autocomplete.js]: Thanks, i already wanted to ask if you could elaborate on this 👍  great catch! I think it's fine to keep it in this PR for now (since it's just a small change).

File: source/skins/simple/raw/FormFieldSkin.js [hunk 2]
```diff
@@ -21,6 +21,9 @@ export default themr(FORM_FIELD)((props) => (
         {props.label}
       </label>
     )}
-    {props.input}
+    <div className={props.theme.inputWrapper}>
+      {props.input}
+      {props.children}
```
Comment (DominikGuzei) [inline source/skins/simple/raw/FormFieldSkin.js]: Could you elaborate on this change?

**Comment by DominikGuzei** [comment]
Yeah i just reviewed the other changes you made regarding the `Options` component and this looks good 👍  At the moment we only need the `Tooltip` additionally (you could do this in a separate PR).

File: source/skins/simple/raw/FormFieldSkin.js (hunk 2 above)
Comment (bendyorke) [reply source/skins/simple/raw/FormFieldSkin.js]: This wraps the action input in a container, allowing item's to be positioned relatively to the actual input, as opposed to the FormField as a whole

**Comment by bendyorke** [comment]
Sounds good.  I just fixed one last bug I noticed where the first element of an `Options#isOpeningUpward` component wouldn't update the arrow colour.  I'll submit a second PR for the Tooltip component!

**Comment by DominikGuzei** [comment]
Thanks for the fixes @bendyorke! Please add an entry to the `CHANGELOG.md`, then i will merge this! 👍

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,17 @@
 Changelog
 =========
 
+## 0.5.6
```
Comment (DominikGuzei) [inline CHANGELOG.md]: It's fine for now (of course you couldn't know), but let's use `## vNext` for the "next unknown version" of the plugin in the changelog section 😉  otherwise it seems like we already released that version!

File: package.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "react-polymorph",
   "description": "React components with highly customizable logic, markup and styles.",
-  "version": "0.5.5",
+  "version": "0.5.6",
```
Comment (DominikGuzei) [inline package.json]: The version here should not be changed in PRs but on `master` right before we publish and tag the next release 😉  please revert this change
